tests/integration/docker/terminal.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+setup() {
+	load common
+	cleanDockerPs
+	runtimeDocker
+}
+
+@test "changing terminal size test" {
+	# This problem is related with https://github.com/01org/cc-oci-runtime/issues/134
+	skip "this functionality is not working"
+	$DOCKER_EXE run -it ubuntu bash -c 'stty rows 50; stty cols 50; if [ `tput cols` != "50" ] && [ `tput lines` != "50" ]; then exit 1; fi'
```

Comment:
Huum, what we want to do is to change the host terminal size and check that the container terminal size matches. That will test that we properly communicate the terminal size to the container.

What's done here is change the terminal size from within the container.